db/schema.rb

```diff
@@ -16,7 +16,7 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
-  create_table "oauth_access_grants", force: true do |t|
+  create_table "oauth_access_grants", force: :cascade do |t|
```

Comment:
Following-up, this seems to be the reason: http://stackoverflow.com/questions/27697119/what-is-cascade-in-rails-schema-rb-and-where-did-it-come-from. I am running Rails 4.2.0 so that seems to make sense.